### ISSUE TITLE
Jest "Styles were unable to be assigned to a file" when using craco.config.js

### ISSUE THREAD

**Issue by cgooderham94** [seek-oss/vanilla-extract#545]
### Describe the bug
When running `npm run test`, tests fail with error:

```
Styles were unable to be assigned to a file. This is generally caused by one of the following:

    - You may have created styles outside of a '.css.ts' context
    - You may have incorrect configuration. See https://vanilla-extract.style/documentation/setup
```
I've seen various issues related to this, where it is suggested that adding the following to `craco.config.js` should work:

```
jest: {
    globals: {
      babelConfig: {
        plugins: ["@vanilla-extract/babel-plugin"],
      },
    },
  },
```
However, this does not resolve the issue.

### Link to reproduction
[My GH repo](https://github.com/cgooderham94/charlie-gooderham-portfolio/tree/feature/skeleton-functionality)

### System Info

```
System:
    OS: macOS 12.1
    CPU: (4) x64 Intel(R) Core(TM) i5-7267U CPU @ 3.10GHz
    Memory: 175.68 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.1 - /usr/local/bin/node
    npm: 8.3.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 96.0.4664.110
    Firefox: 59.0.2
    Safari: 15.2
  npmPackages:
    @vanilla-extract/css: ^1.6.8 => 1.6.8
    @vanilla-extract/webpack-plugin: ^2.1.4 => 2.1.5
```

Thanks in advance! 🙌

**Comment by mattcompiles** [comment]
Honestly not sure what's happening here as it's a big combination of tooling being used together. However, I did check whether the `@vanilla-extract/babel-plugin` was even being required and it wasn't. There must be something wrong with your config. 

While I understand it's frustrating, it's very difficult to debug these kinds of scenarios as the issue isn't with vanilla-extract. Our babel plugin just isn't being applied. I'm going to close this as the issue isn't with vanilla-extract itself, but if you get to the bottom of it feel free to post your solution so other people can benefit from it.

**Comment by cgooderham94** [comment]
Hey @mattcompiles - I'm currently in the process of reverting all packages and setup to what comes out of the box with CRA as this repo is a little outdated and I've picked it up after a few months. Let's see where it gets me.

I really like the concept and principles of vanilla-extract and from what I've experienced, it's really great to use, particularly with the type-safety. I guess the main thing that holds me back from long-term adoption of such un-opinionated tools is the fact that it requires some deviation from CRA template which in itself becomes a bit of a minefield when you start to manually configure your own bundlers/compilers i.e. webpack for such simple projects.

In the case of my repo, the main packages I require are React, TypeScript and Jest (TypeScript CRA template) which is a common and basic setup. I understand it's difficult to debug particular examples and I'm happy to continue investigating. But I wonder if it might be possible to have some examples of common/basic setup requirements such as this in your documentation, for react users? I imagine it would really reduce friction for adoption. In any case, thanks for your quick response on this, enjoy your holidays!
